modules/olm-creating-fb-index-image.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-custom-catalogs.adoc
+
+ifdef::openshift-origin[]
+:registry-image: quay.io/openshift/origin-operator-registry:4.9.0
+endif::[]
+ifndef::openshift-origin[]
+:registry-image: registry.redhat.io/openshift4/ose-operator-registry:v4.9
+endif::[]
+
+[id="olm-creating-fb-index-image_{context}"]
+= Creating a file-based index image
+
+You can create an index image that uses a plain text _file-based catalog_ format (JSON or YAML) instead of the deprecated SQLite database format. The `opm` CLI has new tooling that helps initialize an index in the file-based format, render new records into it, and then validate that the index is valid.
+
+.Prerequisites
+
+* `opm` version 1.18+
+* `podman` version 1.9.3+
+* A bundle image built and pushed to a registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2]
+
+.Procedure
+
+. Initialize an index for a file-based catalog:
+
+.. Create a directory for the index:
++
+[source,terminal]
+----
+$ mkdir <operator_name>-index
+----
+
+.. In the index directory, create a Dockerfile that can build an index image:
```

Comment:
Not in the index directory, the Dockerfile and the index folder are in the same level, as follows:
```console
[cloud-user@preserve-olm-env test]$ ls
etcd-index  etcd-index.Dockerfile
```